vita3k/modules/SceDriverUser/SceAppMgrUser.cpp

```diff
@@ -517,14 +517,16 @@ EXPORT(int, sceAppMgrPspSaveDataRootMount) {
     return UNIMPLEMENTED();
 }
 
-EXPORT(int, sceAppMgrReceiveEvent) {
-    TRACY_FUNC(sceAppMgrReceiveEvent);
+EXPORT(int, sceAppMgrReceiveEvent, SceAppMgrEvent *mgrEvent) {
+    TRACY_FUNC(sceAppMgrReceiveEvent, mgrEvent);
```

Comment:
Is this needed, and shouldn't an error be returned in this case, as setting the event num to 0 in sceAppMgrReceiveEventNum means this function does not have any event to receive

Reply:
yes, you right, i have just set it because is same change but is not obligatory for now